### DISCUSSION TITLE
Reparse for missing data

### DISCUSSION THREAD

**Discussion by BrandonG777** [mealie-recipes/mealie#2211] [Feature Request]
Would like to see a button I click that would reparse the URL for missing data, without overwriting existing data. 

ie. nutritional data was later added to mealie, many of my recipes lack this data but it is available

this would also help for users that have lost images during migrations, like me. ;) 

bonus points for creating a bulk option for this.

**Comment by michael-genson** [comment]
Mealie is at a point where the schema is pretty stable, so I don't think adding this feature would we worth the effort. You could always write a script to do something similar specifically for nutrition info.
